hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/TokenAttributesValidator.java

```diff
@@ -93,6 +103,25 @@ private void tokenStringCheck(
         validateTrue(!s.contains("\u0000"), INVALID_ZERO_BYTE_IN_STRING);
     }
 
+    /**
+     * Given a token byte array, validates that it is not null, not empty, not too long.
+     * @param bytes the byte array to validate
+     * @param maxLen the maximum number of UTF-8 bytes allowed
+     * @param onMissing the response code to use if the bytes array is null or empty
+     * @param onTooLong the response code to use if the bytes array is too long
+     */
+    private void tokenByteArrayCheck(
+            @Nullable final byte[] bytes,
+            final int maxLen,
+            @NonNull final ResponseCodeEnum onMissing,
+            @NonNull final ResponseCodeEnum onTooLong) {
+        validateTrue(bytes != null, onMissing);
+        final int length = bytes.length;
+        validateTrue(length != 0, onMissing);
+        validateTrue(length <= maxLen, onTooLong);
+        validateTrue(!bytes.toString().contains("\u0000"), INVALID_ZERO_BYTE_IN_STRING);
```

Comment:
I would use something like [this](https://github.com/hashgraph/hedera-services/blob/a71e2fbe413bf9c97bfba0d6725f9f75bc2b8a45/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/AttributeValidatorImpl.java#L138) instead

Reply:
Sounds good. Done 👍